packages/node_modules/@webex/plugin-meetings/src/stats/stats.js

```diff
@@ -155,6 +167,20 @@ export default class MeetingStats extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * @param {WebRTCData} data
+   * @returns {undefined}
+   * @public
+   * @memberof MeetingStats
+   */
+  doMetrics(data) {
```

Comment:
DO IT!

Reply:
nit: `data.data` typically drives me nuts

Reply:
i also hate it but it got messed up early on and now it's a decent sized refactor...